v2/spanner-to-sourcedb/src/main/java/com/google/cloud/teleport/v2/templates/transforms/SourceWriterFn.java

```diff
@@ -291,42 +286,43 @@ public void processElement(ProcessContext c) {
         // We need to get and inspect the cause while handling the exception.
       } catch (SpannerException ex) {
         Throwable cause = ex.getCause();
+        String message = ex.getMessage();
+        if (cause != null) {
+          message += ", Caused by: " + cause.getMessage();
+        }
+
         if (cause == null) {
+          // Log all retryable errors. It will be helpful for debugging since the files in retryable
+          // dlq get deleted on consumption.
+          LOG.debug("Retryable error occurred while processing an event: {}", ex.getMessage());
```

Comment:
This will be expensive and can have several unintended side effects as the pipeline scales up - 

1. It can cause the logs to get throttled if the volume of data logged is too high.
2. It can cause OOMs on the worker if the logger is not able to flush the logged data from the memory to the remote client fast enough. 

I suggest we remove this log. We should find alternative ways to root cause events that are being debugged (such as enabling version retention on GCS?)